components/keys/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "keys"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+[dependencies]
+byteorder = "1.2"
+derive_more = "0.15.0"
+hex = "0.3"
+kvproto = { git = "https://github.com/pingcap/kvproto.git" }
+
+tikv_util = { path = "../tikv_util" }
```

Comment:
do we need to depend tikv_alloc here?

Reply:
there aren't any benchmarks in this crate so I don't think so

Reply:
maybe it is better to add it?

Reply:
I have done so.